cowamm/kpi/competitors/all_competitors_4335231.sql

```diff
@@ -0,0 +1,33 @@
+--Groups all competitors in one query
+
+-- Uniswap, PancakeSwap, Sushiwap
+select
+    contract_address,
+    tvl,
+    fee,
+    volume,
+    apr,
+    project,
+    case
+        when '{{blockchain}}' = 'ethereum' then 1
+        when '{{blockchain}}' = 'ethereum' then 100
+        when '{{blockchain}}' = 'arbitrum' then 42161
+    end as chain_id
+from "query_4304295(blockchain='{{blockchain}}')"
+
+union distinct
+
+--Curve
+select
+    contract_address,
```

Comment:
surprised you don't need the token addresses as well here (would make sense imo)

Reply:
Sasha was happy with all the columns in the table...